mobify/progressive-web-scaffold
# Production bundle fixes

If we build for production, some of the optimization/obfuscation steps break how we do routing. 

 **Linked PRs**: #190 

## Changes
- Use the component type instead of component name for routing

## How to test-drive this PR
- Observe the site works as it did previously
- Push a bundle
- Preview the production bundle - observe the site works as it did previously
